test/scripts/test_valhalla_build_config.py

```diff
@@ -0,0 +1,13 @@
+from unittest import TestCase
+
+try:
+    from test.scripts import helper
+except ModuleNotFoundError:
+    import helper
```

Comment:
again the import machinery's trickiness.. this makes sure test can be run from any project folder as working directory.

Reply:
although I don't _quite_ understand why I can run this from e.g. `valhalla/test` directory.. but it does work..